__stories__/Typography.stories.js

```diff
@@ -7,63 +8,74 @@ import {
   H2,
   H3,
   H4,
-  H5,
-  H6,
   Body1,
   Body2,
-  Caption,
-  Overline1,
-  Overline2,
-  Code,
-  InlineCode,
+  StyledButtonText,
+  StyledLinkText,
 } from '../components/ui/Typography'
 
-export const All = () => (
-  <StoryGrid>
-    <div>
-      <H1>Header 1</H1>
-    </div>
-    <div>
-      <H2>Header 2</H2>
-    </div>
-    <div>
-      <H3>Header 3</H3>
-    </div>
-    <div>
-      <H4>Header 4</H4>
-    </div>
-    <div>
-      <H5>Header 5</H5>
-    </div>
-    <div>
-      <H6>Header 6</H6>
-    </div>
-    <div>
-      <Body1>Body 1</Body1>
-    </div>
-    <div>
-      <Body2>Body 2</Body2>
-    </div>
-    <div>
-      <Caption>Caption</Caption>
-    </div>
-    <div>
-      <Overline1>Overline 1</Overline1>
-    </div>
-    <div>
-      <Overline2>Overline 2</Overline2>
-    </div>
-    <div>
-      <Code>Code</Code>
-    </div>
-    <div>
-      <InlineCode>Inline Code</InlineCode>
-    </div>
-  </StoryGrid>
-)
+const linkContainers = {
+  H1,
+  H2,
+  H3,
+  H4,
+  Body1,
+  Body2,
+}
+
+const Template = ({ 'Element Containing Link': linkContainerName }) => {
+  const LinkContainer = linkContainers[linkContainerName]
+  return (
+    <StoryGrid>
+      <div>
+        <H1>Header 1</H1>
+      </div>
+      <div>
+        <H2>Header 2</H2>
+      </div>
+      <div>
+        <H3>Header 3</H3>
+      </div>
+      <div>
+        <H4>Header 4</H4>
+      </div>
+      <div>
+        <Body1>Body 1</Body1>
+      </div>
+      <div>
+        <Body2>Body 2</Body2>
+      </div>
+      <div>
+        <StyledButtonText
+          style={{ backgroundColor: '#4a4a4a', borderColor: '#4a4a4a' }}
```

Comment:
are we planning to use named colors?

Reply:
I guess this is just to show the UI example component in storybook, so, you can skip this.

Reply:
👍 You guessed it. :)